pylxd/container.py

```diff
@@ -70,7 +70,7 @@ def create(cls, client, config, wait=False):
 
     def __init__(self, **kwargs):
         super(Container, self).__init__()
-        for key, value in kwargs.iteritems():
+        for key, value in kwargs.items():
```

Comment:
Hm. This concerns me. I realize what's going on here, but it may be in our best interest to just `import six` and use `six.iteritems` or something equivalent.